From 13c98e7373aa9684f44b09a08b48f2b62c422de5 Mon Sep 17 00:00:00 2001
From: Andrei Stefanescu <andrei.stefanescu@nxp.com>
Date: Tue, 19 Apr 2022 15:22:59 +0300
Subject: [PATCH 27/33] s32: change clk dump header

Change the `clk dump` header to be the same as for the 2022.04 branch.

Issue: ALB-8636

Upstream-Status: Pending 

Signed-off-by: Andrei Stefanescu <andrei.stefanescu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm/mach-s32/s32-cc/scmi_clk_dump.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-s32/s32-cc/scmi_clk_dump.c b/arch/arm/mach-s32/s32-cc/scmi_clk_dump.c
index 4ac863d5d8..997f9f13f3 100644
--- a/arch/arm/mach-s32/s32-cc/scmi_clk_dump.c
+++ b/arch/arm/mach-s32/s32-cc/scmi_clk_dump.c
@@ -129,7 +129,7 @@ int soc_clk_dump(void)
 		return num_clocks;
 	}
 
-	printf(" Rate               Enabled      Name\n");
+	printf(" Rate               Used         Name\n");
 	printf("-------------------------------------------\n");
 
 	for (i = 0; i < num_clocks; ++i) {
-- 
2.17.1

